### PR TITLE
docs: add note to require filter setting for `last_active`

### PR DESCRIPTION
**Pull request by datamweb**
see #646

### CODE REVIEW

File: src/Config/Auth.php [hunk 1]
```diff
@@ -184,6 +184,9 @@ class Auth extends BaseConfig
      * --------------------------------------------------------------------
      * If true, will always update the `last_active` datetime for the
      * logged in user on every page request.
+     * This feature only works when session/tokens/chain filter is set correctly.
```
Comment (kenjis) [inline src/Config/Auth.php:187]: The chain filter does nothing actually, the session/tokens filter does call `recordActiveDate()`.
How about this?
```suggestion
     * This feature only works when session/tokens filter is active.
```

Comment (datamweb) [reply src/Config/Auth.php]: Thanks. Done.

**Comment by datamweb** [comment]
```console
Error: ] Could not process                                                      
         "/home/runner/work/shield/shield/src/Traits/Viewable.php" file, due to:
         "System error: "Call to undefined method                               
         PHPStan\Type\Generic\GenericClassStringType::getObjectClassNames()"    
         Run Rector with "--debug" option and post the report here: https://github.com/rectorphp/rector/issues/new". On line:
         47                                                                     

Error: ] Could not process                                                      
         "/home/runner/work/shield/shield/tests/Unit/CompositionValidatorTest.ph
         p" file, due to:                                                       
         "System error: "Call to undefined method                               
         PHPStan\Type\ObjectWithoutClassType::getObjectClassNames()"            
         Run Rector with "--debug" option and post the report here: https://github.com/rectorphp/rector/issues/new". On line:
         47                                                                     

Error: ] Could not process                                                      
         "/home/runner/work/shield/shield/tests/Unit/DictionaryValidatorTest.php
         " file, due to:                                                        
         "System error: "Call to undefined method                               
         PHPStan\Type\ObjectWithoutClassType::getObjectClassNames()"            
         Run Rector with "--debug" option and post the report here: https://github.com/rectorphp/rector/issues/new". On line:
         47          
```         
see https://github.com/codeigniter4/shield/actions/runs/4240216761/jobs/7369019249

**Comment by datamweb** [comment]
@samsonasik Can you help, please?

**Comment by samsonasik** [comment]
You need to update to latest rector/rector:^0.15.18

**Comment by datamweb** [comment]
@samsonasik Thank you very much, great.🥀

**Review by kenjis — APPROVED** [review]

**Pull request merged**